hi-ml-cpath/src/health_cpath/configs/classification/BaseMIL.py

```diff
@@ -228,6 +229,9 @@ def get_data_module(self) -> HistoDataModule:
     def get_slides_dataset(self) -> Optional[SlidesDataset]:
         return None
 
+    def get_ihc_slides_dataset(self) -> Optional[SlidesDataset]:
+        return None
+
```

Comment:
The naming is very specific to our use case. How about a name like "extra_slides_for_plotting"? This is more generic, and at the same time also indicates clearly that these slides are only used for plotting.

Reply:
Very good point, will change it to extra!

Reply:
I agree!